scout/load/report.py

```diff
@@ -1,33 +1,53 @@
+# -*- coding: utf-8 -*-
 import logging
-import pymongo
 
-from scout.exceptions import (IntegrityError, ValidationError)
+from mongo_adapter import MongoAdapter
+from scout.exceptions import IntegrityError, DataNotFoundError
 
-LOG = logging.getLogger(__name__)
+logger = logging.getLogger(__name__)
+
+
+def load_delivery_report(adapter: MongoAdapter,
+                         report_path: str,
+                         case_id: str,
+                         update: bool = False):
+    """Load a delivery report into the database
+
+    If the report already exists the function will exit.
+    If the user want to load a report that is already in the database
+    'update' has to be 'True'
 
-def load_report(adapter, case_id, report_path, update=False):
-    """Add the path to a report to a case
-    
     Args:
```

Comment:
Please use [napoleon](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) style on docstrings which is used through the whole package. Also mentioned in [development](https://github.com/Clinical-Genomics/development/blob/6942b35731c169b49f67500a57bd75afa050cb0b/documentation/README.md)

Reply:
I get your general idea, use napoleon style on docstrings. But what specifically is it that you want? I copied this docstring from another method, was it napoleon or should it have been napoleon, I don't follow.

Reply:
I presume Mans means to remove the `:param` from the statements, e.g.

`:param report_path: (string) path to report ` -> `report_path (string): path to report`

Reply:
Fixed :)